baker/algolia/indexExplorersToAlgolia.ts

```diff
@@ -120,17 +121,22 @@ const getExplorerRecords = async (
 
     // Fetch info about all charts used in explorers, as linked by the explorer_charts table
     const graphersUsedInExplorers = await db
-        .knexRaw<{ chartId: number }>(
+        .knexRaw<DbInsertChart>(
             `
-        SELECT DISTINCT chartId
-        FROM explorer_charts
+        SELECT * FROM charts
+        INNER JOIN (
+            SELECT DISTINCT chartId AS id FROM explorer_charts
+        ) AS ec
+        USING (id)
         `,
             knex
         )
-        .then((results: { chartId: number }[]) =>
-            results.map(({ chartId }) => chartId)
+        .then((charts) =>
+            charts.map(
+                (c) =>
+                    ({ ...c, config: JSON.parse(c.config) }) as DbEnrichedChart
```

Comment:
This works but you can also use the `parseChartsRow`. As a convention we have such parse* and serialize* functions for all entities that have fields that need to be parsed and they already have the right type signature filled in